apps/wfo-ui/utils/queryVarsMappers.ts

```diff
@@ -14,9 +14,9 @@ export const mapSortOrderToGraphQlSortOrder = (
 
 export const mapToGraphQlSortBy = <Type>(
     sortBy: GraphQLSort<Type> | undefined,
-): GraphqlSort | null => {
+): GraphqlSort | undefined => {
```

Comment:
If sortBy in packages/orchestrator-ui-components/src/hooks/useDataDisplayParams.ts is always defined, we might want to update this function as well. Looks like the if statement might not be needed, since sortBy, sortBy.order and sortBy.field will not be undefined.

Reply:
Yes. Removed the undefined and the if